packer/epoch.json

```diff
@@ -5,6 +5,15 @@
   },
   "builders":
   [
+    {
+      "type": "googlecompute",
+      "account_file": "/tmp/epoch-p2p.json",
```

Comment:
I see acount_file documented in https://www.packer.io/docs/builders/googlecompute.html

This hardcoded file may surprise user though seems ok. May be worth mention in README.md#credentials-setup